packages/aws-client/src/StepFunctions.ts

```diff
@@ -84,46 +95,48 @@ export const executionExists = (executionArn: string) =>
  * If a ThrottlingException is received, this function will retry using an
  * exponential backoff.
  *
- * @param {Object} params
- * @returns {Promise<Object>}
+ * @param {GetExecutionHistoryInput} params
+ * @returns {Promise<GetExecutionHistoryOutput>}
  *
  * @kind function
  */
-export const getExecutionHistory = improveStackTrace(
-  retryOnThrottlingException(
-    async (
-      params: AWS.StepFunctions.GetExecutionHistoryInput,
-      previousResponse: { events: AWS.StepFunctions.HistoryEventList } = {
-        events: [],
-      }
-    ): Promise<{ events: AWS.StepFunctions.HistoryEventList }> => {
-      const response = await sfn().getExecutionHistory(params).promise();
-      const events = [
-        ...previousResponse.events,
-        ...response.events,
-      ];
-      // If there is a nextToken, recursively call this function to get all events
-      // in the execution history.
-      if (response.nextToken) {
-        return getExecutionHistory({
-          ...params,
-          nextToken: response.nextToken,
-        }, {
-          events,
-        });
-      }
-      return {
+export const getExecutionHistory = retryOnThrottlingException(
```

Comment:
Same question re: `improveStackTrace`. I think all of the formatting changes here are just updating the indenting because we removed this call.

Reply:
Per Charles and Jenny, v3 has better stack track built in, so they have been removing "improveStackTrace" in their v3 updates.